app/src/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+const Dashboard = ({ score, handlePitch, handleReset }) => {
+  const disabled = {
+    background: `rgba(0, 0, 0, 0.1)`,
+    cursor: 'not-allowed'
+  }
+  const handleClick = e => {
+    const name = e.target.name
+    handlePitch(name)
+  }
+  return (
+    <div className="buttonContainer">
+      <div
+        className="button"
+        name="hit"
+        style={score.struckOut || score.walked || score.hit ? disabled : null}
+        onClick={
+          score.struckOut || score.walked || score.hit ? null : handleClick
+        }>
+        Hit
+      </div>
+      <div
+        className="button"
+        name="foul"
+        style={score.struckOut || score.walked || score.hit ? disabled : null}
+        onClick={
+          score.struckOut || score.walked || score.hit ? null : handleClick
+        }>
+        Foul
+      </div>
+      <div
+        className="button"
+        name="strike"
+        style={score.struckOut || score.walked || score.hit ? disabled : null}
+        onClick={
+          score.struckOut || score.walked || score.hit ? null : handleClick
+        }>
+        Strike
+      </div>
+      <div
+        className="button"
+        name="ball"
+        style={score.struckOut || score.walked || score.hit ? disabled : null}
+        onClick={
+          score.struckOut || score.walked || score.hit ? null : handleClick
```

Comment:
You could reduce repetition in the `style`s and `onClick`s by handling them before the return:
```javascript
  const style = {score.struckOut || score.walked || score.hit ? disabled : null}
  const onClick = {
    score.struckOut || score.walked || score.hit ? null : handleClick
  }

  return (
    <div className="buttonContainer">
      <div
        className="button"
        name="hit"
        style={style}
        onClick={onClick}
      >
        Hit
      </div>
      <div
        className="button"
        name="foul"
        style={style}
        onClick={onClick}
      >
        Foul
      </div>
    // ...
  )
```
At that point, these buttons might as well just become a component:
```javascript
const Button = ({ name }) => (
  <div
    className="button"
    name={name.toLowerCase()}
    style={style}
    onClick={onClick}
  >
    {name}
  </div>
);
```
with a usage like:
```javascript
  return (
    <div className="buttonContainer">
      <Button name={'Hit'} />
      <Button name={'Foul'} />
    // ...
  )